src/frontend_analysis/IR_manipulation/create_tree_manager.cpp

```diff
@@ -392,7 +392,6 @@ DesignFlowStep_Status create_tree_manager::Exec()
       const auto fnode = TM->GetFunction(symbol);
       if(!fnode)
       {
-         THROW_WARNING("Function specified in architecture XML is missing in the IR: " + symbol);
```

Comment:
Yes, but actually no 😄

Reply:
In complex designs, this message is very annoying. Maybe it is just a matter of putting it under a high verbose level.